definitions/variable/technology/electricity-operation.yaml

```diff
@@ -108,6 +108,9 @@
 - Import|Electricity:
     description: Exported electricity from a given region
     unit: MWh
+- Flow|Electricity|{Region}|{Region}:
```

Comment:
```suggestion
- Flow|Electricity:
```

Reply:
the column region is meant for describing the region this variable applies to. Directionnal regions are already included and have to be written region1 > region2 (in the region column), which here means that the energy flow is from region1 to region2 (can be negative which means the flow is from region2 to region1)

Reply:
Thanks for the quick review and the info, I didn't know about directional regions. I'll commit your suggestion.